op-challenger/game/keccak/matrix/matrix_test.go

```diff
@@ -220,6 +195,126 @@ func TestMatrix_AbsorbNextLeaf(t *testing.T) {
 	}
 }
 
+func TestVerifyPreimage(t *testing.T) {
+	preimage := testutils.RandomData(rand.New(rand.NewSource(2323)), 1024)
+	validCommitments := func() []common.Hash {
+		s := NewStateMatrix()
+		valid, err := s.AbsorbUpTo(bytes.NewReader(preimage), 1000*types.BlockSize)
```

Comment:
Below we inline this like so. Not sure if the operation was split for formatting.
```suggestion
		valid, err := NewStateMatrix().AbsorbUpTo(bytes.NewReader(preimage), 1000*types.BlockSize)
```